README.md

```diff
@@ -27,7 +27,7 @@ Code examples for each language's SDK can be found within the following subdirec
 | PHP        | [php/](php)              | 3           |
 | Python     | [python/](python)           | 3           |
 | Ruby       | [ruby/](ruby)             | 3           |
-| Rust       | [rust_dev_preview/](rust_dev_preview) | preview             |
+| Rust       | [rustv1/](rustv1) | preview             |
```

Comment:
Is `preview` in the far right column meant to be 1?

Reply:
Good catch, thank you